### DISCUSSION TITLE
Unable to load file into Doris table in external catalog

### DISCUSSION THREAD

**Discussion by nikola-arcadis** [apache/doris#40207] [A - General / Q&A]
I have an external catalog in Doris using Iceberg REST. SQL insert and select works just fine.

but when I want to load a file from s3 into iceberg table I get strange error.

This is my configuration:

```
LOAD LABEL s3_csv_load (
  DATA INFILE("s3://warehouse/tasks.csv") INTO TABLE tasks COLUMNS TERMINATED BY ","
) WITH S3 (
  "AWS_ENDPOINT" = "...",
  "AWS_ACCESS_KEY" = "...",
  "AWS_SECRET_KEY" = "...",
  "AWS_REGION" = "...",
  "use_path_style" = "true"
) PROPERTIES (
  "timeout" = "3600"
);
```
and when I execute I get this

```
MySQL Error
Unable to connect to database "tasks_db".
```

The database clearly exists.

Interestingly enough if I run this import into internal table it works fine.

**Comment by ixzc** [comment]
this is not support, you can use tvf and insert into

**Comment by nikola-arcadis** [comment]
That works, thanks, but I'm where this process happens? Using broker, the processing is offloaded to a broker instance which can scale independently. This seems like a performance hit to backend nodes.
